public/docs/ts/latest/glossary.jade

```diff
@@ -188,28 +186,24 @@ block includes
     Applications display data values to a user and respond to user
     actions (clicks, touches, keystrokes).
 
-    We could push application data values into HTML, attach
-    event listeners, pull changed values from the screen, and
-    update application data values ... all by hand.
-
-    Or we could declare the relationship between an HTML widget
-    and an application data source ... and let a data binding
+    Instead of manually pushing application data values into HTML, attaching
+    event listeners, pulling changed values from the screen, and
+    updating application data values, you can use data binding by declaring the relationship between an HTML widget and data source and let the
     framework handle the details.
 
-    Data Binding is that second approach. Angular has a rich
-    data binding framework with a variety of data binding
+    Angular has a rich data binding framework with a variety of data binding
     operations and supporting declaration syntax.
 
      The many forms of binding include:
-     * [Interpolation](!{docsLatest}/guide/template-syntax.html#interpolation)
-     * [Property Binding](!{docsLatest}/guide/template-syntax.html#property-binding)
-     * [Event Binding](!{docsLatest}/guide/template-syntax.html#event-binding)
-     * [Attribute Binding](!{docsLatest}/guide/template-syntax.html#attribute-binding)
-     * [Class Binding](!{docsLatest}/guide/template-syntax.html#class-binding)
-     * [Style Binding](!{docsLatest}/guide/template-syntax.html#style-binding)
-     * [Two-way data binding with ngModel](!{docsLatest}/guide/template-syntax.html#ng-model)
-
-    Learn more about data binding in the
+     * [Interpolation](!{docsLatest}/guide/template-syntax.html#interpolation).
+     * [Property Binding](!{docsLatest}/guide/template-syntax.html#property-binding).
+     * [Event Binding](!{docsLatest}/guide/template-syntax.html#event-binding).
+     * [Attribute Binding](!{docsLatest}/guide/template-syntax.html#attribute-binding).
+     * [Class Binding](!{docsLatest}/guide/template-syntax.html#class-binding).
+     * [Style Binding](!{docsLatest}/guide/template-syntax.html#style-binding).
+     * [Two-way data binding with ngModel](!{docsLatest}/guide/template-syntax.html#ng-model).
+
+    Read more about data binding in the
```

Comment:
Changed the intro language to indicate the Template Syntax page. I can change the language in each bullet point but since they all go to the same page, I've put it up top.